Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -523,8 +538,37 @@ namespace gdjs {
     getLastStickInputAngle() {
       return this._stickAngle;
     }
+
+    registerHook(
```

Comment:
Maybe add a unregisterHook for completeness?

Reply:
I did but removed it. Object instances should have the same hooks on their whole lifetime. I think it would be a bad idea to unregister when the other behavior is deactivated for instance.

This comment in gd.js comforted me in this decision.
```
  /**
   * Unregister a callback.
   * This should not be used apart from the code generated from extensions
   * events functions, to handle hot-reloading.
   * In any other case, a callback should be registered once, and only once.
   *
   * @internal
   */
  export const _unregisterCallback = function (callback: unknown): void {
```

Reply:
Do you think I should still add one with a warning in the documentation?

Reply:
This is very fair, let's not add a "unregisterHook" as indeed we want to avoid this "dynamicness".
Let's still add a warning explaining this :)